packages/manager/modules/support/src/new-ticket/issues-form/issues-form.controller.js

```diff
@@ -1,14 +1,16 @@
-import constants from './issues-form.constant.js';
+import constants from './issues-form.constant';
 
 export default class SupportNewIssuesFormController {
   /* @ngInject */
-  constructor($q, $state, $translate, OvhApiService, OvhApiSupport, OvhApiMe) {
+  constructor($q, $state, $translate, OvhApiService,
```

Comment:
```suggestion
  constructor(
    $q,
    $state,
    $translate,
    OvhApiService,
    OvhApiSupport,
    OvhApiMe,
    SupportNewTicketService,
  )
```
makes it easier to add injections imo :)